### PR TITLE
Upgrade to VS 2017

### PR DESCRIPTION
**Pull request by natemcmaster**
Removes project.json and upgrades to MSBuild.

### CODE REVIEW

File: src/Microsoft.Extensions.Logging.Abstractions/Microsoft.Extensions.Logging.Abstractions.csproj [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  <Import Project="..\..\build\common.props" />
+
+  <PropertyGroup>
+    <Description>Logging abstractions for Microsoft.Extensions.Logging.
+Commonly used types:
+Microsoft.Extensions.Logging.ILogger
+Microsoft.Extensions.Logging.ILoggerFactory
+Microsoft.Extensions.Logging.ILogger&lt;TCategoryName&gt;
+Microsoft.Extensions.Logging.LogLevel
+Microsoft.Extensions.Logging.
```
Comment (BrennanConroy) [inline src/Microsoft.Extensions.Logging.Abstractions/Microsoft.Extensions.Logging.Abstractions.csproj]: This doesn't look right

File: test/Microsoft.Extensions.Logging.Testing.Tests/Microsoft.Extensions.Logging.Testing.Tests.csproj [hunk 2]
```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\build\common.props" />
+
+  <PropertyGroup>
+    <VersionPrefix>1.1.0</VersionPrefix>
```
Comment (BrennanConroy) [inline test/Microsoft.Extensions.Logging.Testing.Tests/Microsoft.Extensions.Logging.Testing.Tests.csproj]: Get rid of this?

**Comment by BrennanConroy** [comment]
Minor comments
:shipit:

File: src/Microsoft.Extensions.Logging.Abstractions/Microsoft.Extensions.Logging.Abstractions.csproj (hunk 1 above)
Comment (natemcmaster) [inline src/Microsoft.Extensions.Logging.Abstractions/Microsoft.Extensions.Logging.Abstractions.csproj]: Huh, yeah, looks wrong, but it's been that way for almost a year. cc @Eilon who wrote this description?

image

Comment (natemcmaster) [inline src/Microsoft.Extensions.Logging.Abstractions/Microsoft.Extensions.Logging.Abstractions.csproj]: I've updated to make this look better.

Comment (Eilon) [inline src/Microsoft.Extensions.Logging.Abstractions/Microsoft.Extensions.Logging.Abstractions.csproj]: I accept the blame for that; thanks for fixing 😄

**Pull request merged**